sdk/service/sdk.go

```diff
@@ -44,6 +47,9 @@ func (s *SDK) Create(req *api.CreateServiceRequest, accountName, accountPassword
 	msg := newMsgCreateService(s.cdc, req, owner)
 	tx, err := s.client.BuildAndBroadcastMsg(msg, accountName, accountPassword, accNumber, accSeq)
 	if err != nil {
+		if err == mempool.ErrTxInCache {
+			return nil, errors.New("service already exists")
```

Comment:
what about wrapping the original error? I would like keep it to not forget it in the future. I see this as a small hack to help user understand what is happening. But as a developer, i want to know the tx already exists.
```suggestion
			return nil, fmt.Error("service already exists: %w", err)
```